pandas/tests/io/test_parquet.py

```diff
@@ -640,6 +641,7 @@ def test_use_nullable_dtypes(self, engine, request):
             expected = expected.drop("c", axis=1)
         tm.assert_frame_equal(result2, expected)
 
+    @pytest.mark.skipif(pa_version_under6p0, reason="minimum pyarrow not installed")
```

Comment:
Doesn't `engine` already do a similar `skipif`?

Reply:
I thought so as well, but it was failing locally without pyarrow

Reply:
Ah sorry, actually it's only skipped for pyarrow engine but it is imported for both engines, e.g. the test will fail if fast parquet engine is used